app/views/settings/index.html.erb

```diff
@@ -29,6 +29,13 @@
         <h6 class="text-center">Copy & Paste the following into your app:</h6>
         <pre class="text-center">http://localhost:3000/api/v1/links/create?api_key=<%= @settings.api_key %>&url=%@</pre>
       </div>
+      <div class="panel-body">
+        <%= form_for :settings, url: {action: "update_api_key"}, method: :put, html: { class: 'form-horizontal' } do |f| %>
+          <div class="actions">
+            <%= f.submit "Update API Key", class: "btn-lg btn-primary btn-block" %>
```

Comment:
To have a popup to verify the user meant to reset their key, you can say:

`f.submit "Update API Key", data: {confirm: "Are you sure?  This will obsolete previous keys."} `